chat/src/main/java/org/exoplatform/platform/qa/ui/chat/pageobject/MessageManagement.java

```diff
@@ -0,0 +1,82 @@
+package org.exoplatform.platform.qa.ui.chat.pageobject;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Configuration;
+import org.exoplatform.platform.qa.ui.selenium.TestBase;
+
+import static com.codeborne.selenide.Selectors.*;
+import static com.codeborne.selenide.Selenide.*;
+import static junit.framework.TestCase.assertEquals;
+import static org.exoplatform.platform.qa.ui.selenium.locator.chat.ChatLocator.*;
+
+public class MessageManagement {
+    private final TestBase testBase;
+
+    public MessageManagement(TestBase testBase) {
+        this.testBase = testBase;
+    }
+
+    public void checkMessageMenu(String message){
+
+        ELEMENT_CHAT_LIST_MSG.find(byText(message)).hover();
+        ELEMENT_CHAT_LIST_MSG.find(byClassName("uiIconDots")).click();
```

Comment:
you must specify which message you want to open its drop down menu

Reply:
done